hack/olm-registry/olm-artifacts-template-hive.yaml

```diff
@@ -0,0 +1,268 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: selectorsyncset-template
+
+parameters:
+- name: REGISTRY_IMG
+  required: true
+- name: CHANNEL
+  required: true
+- name: IMAGE_TAG
+  required: true
+- name: IMAGE_DIGEST
+  requred: true
+- name: REPO_NAME
+  value: aws-vpce-operator
+  required: true
+- name: DISPLAY_NAME
+  value: AWS VPCE Operator Acceptance Controller
+  required: true
+
+objects:
+###############
+# FedRAMP SSS #
+###############
+- apiVersion: hive.openshift.io/v1
+  kind: SelectorSyncSet
+  metadata:
+    annotations:
+      component-display-name: ${DISPLAY_NAME}
+      component-name: ${REPO_NAME}
+      telemeter-query: csv_succeeded{_id="$CLUSTER_ID",name=~"${REPO_NAME}.*",exported_namespace=~"openshift-.*",namespace="openshift-operator-lifecycle-manager"} == 1
+    labels:
+      managed.openshift.io/gitHash: ${IMAGE_TAG}
+      managed.openshift.io/gitRepoName: ${REPO_NAME}
+      managed.openshift.io/osd: 'true'
+    name: aws-vpce-operator-acceptance
+  spec:
+    clusterDeploymentSelector:
+      matchLabels:
+        api.openshift.com/managed: 'true'
+        api.openshift.com/fedramp: 'true'
+        api.openshift.com/private-link: 'true'
```

Comment:
Is the Hive-targeting portion happening elsewhere? This would still deploy the SSS to all clusters in FedRAMP

Reply:
My intention was to be able to deploy it to more than just hive clusters, I want to validate a different versions and configurations of the entire acceptance controller workflow in lower environments on both hive and non-hive clusters without needing to promote AVO in upper environments that own lower environments.